en/simulation/ignition_gazebo.md

```diff
@@ -56,12 +56,114 @@ In order to run the simulation without running the ignition gazebo gui, one can
 HEADLESS=1 make px4_sitl gz_x500
 ```
 
-In order to increase the verbose output, `VERBOSE_SIM=1` can be used:
+## Usage/Configuration Options
+
+The startup pipeline allows for highly flexible configuration.
+In particular, it is possible to:
+
+- Start a new Ignition simulation with an arbitrary world or attach to an already running simulation.
+- Add a new vehicle to Ignition or link a new PX4 instance to an existing one.
+
+These scenarios are managed by setting the appropriate environment variables.
+
+### Syntax
+
+The startup syntax takes the form:
 
 ```bash
-VERBOSE_SIM=1 make px4_sitl gz_x500
+ARGS ./build/px4_sitl_default/bin/px4
 ```
 
+where `ARGS` is a list of environment variables including:
+
+- `PX4_SYS_AUTOSTART` (_mandatory_):
+  Sets the [airframe autostart id](../dev_airframes/adding_a_new_frame.md) of the PX4 airframe to start.
+  Only `4001` (x500 quadcopter) is currently supported.
+
+- `PX4_GZ_WORLD` (_optional_):
+  Sets the Ignition world file for a new simulation.
+  If it is not given, then [default](https://github.com/PX4/PX4-Autopilot/blob/main/Tools/simulation/gz/worlds/default.sdf) is used.
+  This variable is ignored if an existing simulation is already running.
+
+- `PX4_GZ_MODEL_NAME`:
```

Comment:
Should this and the model also be marked as optional?

Reply:
Actually, yes. If they are both omitted then PX4 throws a warning and then uses `PX4_SIM_MODEL` as model name in the exact same way it uses `PX4_GZ_MODEL` (but then `PX4_GZ_MODEL_POSE` is completely ignored, even if given). That's why I chose to not mention this possibility. `PX4_SIM_MODEL` should only be used when starting the simulation with `make px4_sitl gz_x500` as its default value is defined in the airframe file. I think that @bperseghetti kept it only for that reason.

Reply:
Taking a further look I decided to be more precise about the default variables and the role of `PX4_SIM_MODEL`. I hope that it is not too messy now.

Reply:
It's better. Though of course I couldn't help myself mixing this around a bit too.